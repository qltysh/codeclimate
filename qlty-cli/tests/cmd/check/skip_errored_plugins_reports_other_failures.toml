bin.name = "qlty"
args = ["check", "--all", "--no-cache", "--skip-errored-plugins"]
status.code = 1
