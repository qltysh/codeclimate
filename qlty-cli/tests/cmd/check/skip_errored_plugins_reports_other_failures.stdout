
 ISSUES 

:0:0
    0:0  high    always_fail failed  always_fail:fail

Lint error always_error: Exited with code 1 .qlty/out/invocations/invoke-[..].yaml

